pkg/kapp/cmd/appchange/list.go

```diff
@@ -75,14 +110,18 @@ func (t AppChangesTable) Print(ui ui.UI) {
 			uitable.NewHeader("Description"),
 			nsHeader,
 		},
-
```

Comment:
nit: I think we can continue having this new line.

Reply:
added back.